blogposts/from-saas-to-on-prem.md

```diff
@@ -0,0 +1,54 @@
+In mid 2017, we decided to go against the SaaS trend and turn our product into an on-prem, self-hosted solution.
+
+## The transition from SaaS to On-prem
+While a decade ago on-premises solutions were difficult to setup and required on-site installation, Continuous Integration (CI) has made it possible to verify builds in different environments and increase successful deployments off-site. [Docker](https://docs.sourcegraph.com/admin/install/docker) (for shipping) and [Kubernetes](https://github.com/sourcegraph/deploy-sourcegraph) (for container orchestration) simplify packaging up software and running it on [AWS](https://docs.sourcegraph.com/admin/install/docker/aws), [Google Cloud](https://docs.sourcegraph.com/admin/install/docker/google_cloud) or [DigitalOcean](https://docs.sourcegraph.com/admin/install/docker/digitalocean). This infrastructure eased the way for Sourcegraph to transition from a Software as a Service to a self-hosted product.
+
+To make it simple for our customers to stay on the latest Sourcegraph version, we optimized the updating process with the following changes:
+
+- We moved from continuous delivery to monthly iteration cycles.
+- We emphasized feature development to encourage keeping up with the newest Sourcegraph version. We also have policies that ask customers to stay on the newest versions.
+- We established a standard release testing process to better detect bugs in our releases.
+- We wrote [better docs](https://docs.sourcegraph.com/admin/install) to tackle the challenge of running Sourcegraph smoothly on different environments.
+
+## Code Base Security
+Many of our customers have high security requirements and don’t feel comfortable uploading their private code to an external cloud software provider. They prefer a solution that can run on their own infrastructure, under their own security terms and authorization mechanisms, with full control over who has access. Since the release of Sourcegraph 2.3 as an on-premises, self-hosted product, we’ve experienced much higher Sourcegraph adoption rates for enterprise usage.
+
+![Weekly Active Users from Customers since 2017](images/on-prem-customerWAUs.jpg "Weekly Active Users from Customers since 2017")
```

Comment:
Can we change the title of this to "Weekly Active Users from paying customers since 2017"?

I.e., sentence case, and add "paying" in... It's redundant, but I think stressing it here might be helpful